tests/replicator.rs

```diff
@@ -39,12 +39,24 @@ fn test_replicator_startup() {
         tmp_copy_ledger(&leader_ledger_path, "replicator_test_validator_ledger");
 
     {
+        let (signer, t_signer, signer_exit) = local_vote_signer_service().unwrap();
+        let rpc_client = RpcClient::new_from_socket(signer);
+
+        let msg = "Registering a new node";
```

Comment:
is this magic string used elsewhere, or is this just something concocted for testing?

Reply:
This string is a placeholder while registering new nodes with the voting service. for now. It's signed by the client and signature is verified by the signer service. Need to use a better "msg" than this, that cannot be replayed.